packages/flutter_markdown/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.6.11
```

Comment:
Since this is a bugfix, it should be 0.6.11+3.

Reply:
I guess you meant `0.6.10+3`. Fixed!